Python -O is completely useless, lets reduce the amount of python stuff we ship

--- rpm-4.12.0.1/scripts/brp-python-bytecompile~	2014-06-30 04:47:14.000000000 -0400
+++ rpm-4.12.0.1/scripts/brp-python-bytecompile	2015-07-18 19:50:54.794600309 -0400
@@ -48,11 +48,11 @@
 	fi
 
 	# Generate optimized (.pyo) byte-compiled files.
-	$python_binary -O -c 'import compileall, sys; sys.exit(not compileall.compile_dir("'"$python_libdir"'", '"$depth"', "'"$real_libdir"'", force=1, quiet=1))'
-	if [ $? -ne 0 -a 0$errors_terminate -ne 0 ]; then
-		# One or more of the files had a syntax error
-		exit 1
-	fi
+	#$python_binary -O -c 'import compileall, sys; sys.exit(not compileall.compile_dir("'"$python_libdir"'", '"$depth"', "'"$real_libdir"'", force=1, quiet=1))'
+	#if [ $? -ne 0 -a 0$errors_terminate -ne 0 ]; then
+	#	# One or more of the files had a syntax error
+	#	exit 1
+	#fi
 done
 
 
@@ -67,9 +67,10 @@
 fi
 
 # Generate optimized (.pyo) byte-compiled files.
-$default_python -O -c 'import compileall, re, sys; sys.exit(not compileall.compile_dir("'"$RPM_BUILD_ROOT"'", '"$depth"', "/", 1, re.compile(r"'"/bin/|/sbin/|/usr/lib(64)?/python[0-9]\.[0-9]"'"), quiet=1))' > /dev/null
-if [ $? -ne 0 -a 0$errors_terminate -ne 0 ]; then
-	# One or more of the files had a syntax error
-	exit 1
-fi
+#$python2 --c 'import compileall, re, sys; sys.exit(not compileall.compile_dir("'"$RPM_BUILD_ROOT"'", '"$depth"', "/", 1, re.compile(r"'"/bin/|/sbin/|/usr/lib(64)?/python2[0-9]\.[0-9]"'"), quiet=1))' > /dev/null
+#$python3 --c 'import compileall, re, sys; sys.exit(not compileall.compile_dir("'"$RPM_BUILD_ROOT"'", '"$depth"', "/", 1, re.compile(r"'"/bin/|/sbin/|/usr/lib(64)?/python3[0-9]\.[0-9]"'"), quiet=1))' > /dev/null
+#if [ $? -ne 0 -a 0$errors_terminate -ne 0 ]; then
+#	# One or more of the files had a syntax error
+#	exit 1
+#fi
 exit 0
--- rpm-4.12.0.1//scripts/brp-python-hardlink~	2014-06-30 08:47:14.000000000 +0000
+++ rpm-4.12.0.1//scripts/brp-python-hardlink	2017-10-19 10:09:23.703125859 +0000
@@ -5,6 +5,8 @@
 	exit 0
 fi
 
+exit 0
+
 # Hardlink identical *.pyc and *.pyo, originally from PLD's rpm-build-macros
 # Modified to use sha1sum instead of cmp to avoid a diffutils dependency.
 find "$RPM_BUILD_ROOT" -type f -name "*.pyc" | while read pyc ; do
